json_schemas/buyer-email-domains.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "additionalProperties": false,
+  "properties": {
+    "domainName": {
+      "pattern": "^(?!:\/\/)([a-zA-Z0-9-_]+\\.)*[a-zA-Z0-9][a-zA-Z0-9-_]+\\.[a-zA-Z]{2,64}?$",
+      "type": "string"
+    }
+  },
+  "required": ["domainName"],
+  "title": "Create Buyer Domain Validation Schema",
```

Comment:
Could just be `Buyer Domain Schema` - we don't mention the action ("Create") or validation in any of our other schema titles.

Reply:
Cool - now done.